src/module/deadman-switch/installation.ts

```diff
@@ -0,0 +1,20 @@
+import { Address, encodePacked } from 'viem'
+import { Module } from '../types'
+import { DEADMAN_SWITCH_ADDRESS } from './constants'
+
```

Comment:
just realised I dont think we added this for the other modules, but do you think it'd be useful to add a getter function, in this case for the config? Imo this could be useful for a dapp or wallet to display to the user what config is active atm